R/covariance_plot.R

```diff
@@ -1,4 +1,6 @@
-#' Helpers for Covariance Plot
+#' Time points helper for Covariance Plot
+#'
+#' @description `r lifecycle::badge("experimental")`
```

Comment:
we can / should omit this here now because the function is internal, i.e. not exported
```suggestion
```